examples/compute_polar_coordinates.py

```diff
@@ -0,0 +1,362 @@
+"""Express 2D vectors in polar coordinates
+============================================
+
+Compute a vector representing head direction and express it in polar
+coordinates.
+"""
+
+# %%
+# Imports
+# -------
+
+# For interactive plots: install ipympl with `pip install ipympl` and uncomment
+# the following line in your notebook
+# %matplotlib widget
+import numpy as np
+from matplotlib import pyplot as plt
+
+from movement import sample_data
+from movement.io import load_poses
+from movement.utils.vector import cart2pol, pol2cart
+
+# %%
+# Load sample dataset
+# ------------------------
+# In this tutorial, we will use a sample dataset with a single individual
+# (a mouse) and six keypoints.
+
+ds_path = sample_data.fetch_dataset_paths(
+    "SLEAP_single-mouse_EPM.analysis.h5"
+)["poses"]
+ds = load_poses.from_sleap_file(ds_path, fps=None)  # force time_unit = frames
+
+print(ds)
+print("-----------------------------")
+print(f"Individuals: {ds.individuals.values}")
+print(f"Keypoints: {ds.keypoints.values}")
+
+
+# %%
+# The loaded dataset ``ds`` contains two data variables:``position`` and
+# ``confidence``. Both are stored as data arrays. In this tutorial, we will
+# use only ``position``:
+position = ds.position
+
+
+# %%
+# Compute head vector
+# ---------------------
+# To demonstrate how polar coordinates can be useful in behavioural analyses,
+# we will compute the head vector of the mouse.
+#
+# We define it as the vector from the midpoint between the ears to the snout.
+
+# compute the midpoint between the ears
+midpoint_ears = 0.5 * (
+    position.sel(keypoints="left_ear") + position.sel(keypoints="right_ear")
+)
+
+# compute the head vector
+head_vector = position.sel(keypoints="snout") - midpoint_ears
+
+# drop the keypoints dimension
+head_vector = head_vector.drop_vars("keypoints")
```

Comment:
it's not immediately clear why this is necessary, maybe a short explanation is warranted?

Reply:
Actually, I think there is a better, more "xarray"ic way to computed the midpoint, which will automatically get rid of the "keypoints" dim for you:
```python
midpoint_ears = position.sel(keypoints=["left_ear", "right_ear"]).mean(dim="keypoints")
```
Here, because we are explicitly taking the mean across the keypoints, xarray smartly collapses that dimension.

Reply:
Oh but I realised, once you subtract the midpoint from the snout, the "keypoints" dim sneaks back in.
To avoid that, you could do:
```python
head_vector = position.sel(keypoints="snout").values - midpoint_ears
```
Though, this is necessarily more transparent than your version, so feel free to choose whichever you like best.

Reply:
thanks! I like that we show `xarray` methods so went for the following
- using the `mean()` approach
- using `drop_vars()`, but also added more context in the comment.